docs/Introduction.stories.mdx

```diff
@@ -1,7 +1,7 @@
-import { Meta, Description } from '@storybook/addon-docs';
+import { Meta, Markdown } from '@storybook/addon-docs';
 
 import README from '../README.md';
 
 <Meta title="Getting Started/Introduction" />
 
-<Description markdown={README} />
+<Markdown>{README}</Markdown>
```

Comment:
The `markdown` prop has been deprecated in favor of the <Markdown> component https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#description-block-parametersnotes-and-parametersinfo